### PR TITLE
Removed TabletStatistics.getSplitCreationTime SPI method

### LINKED ISSUE #3595: Remove TabletStatistics.getSplitCreationTime() from balancer SPI
Noticed this while working on #3592.  The bit of information that was passed to the balancer was based on if  tablet recently split on a tablet server.  Tablet no longer split on tablet servers, so this information is no longer relevant.

### PR DESCRIPTION
**Pull request by dlmarion**
Fixes #3595

### CODE REVIEW

**Comment by dlmarion** [comment]
I will create a follow-on task on merge to mark this method as deprecated in main.

File: core/src/main/java/org/apache/accumulo/core/spi/balancer/SimpleLoadBalancer.java [hunk 1]
```diff
@@ -265,7 +265,10 @@ List<TabletMigration> move(ServerCounts tooMuch, ServerCounts tooLittle, int cou
         log.error("Unable to select a tablet to move", ex);
         return result;
       }
-      TabletId tabletId = selectTablet(onlineTabletsForTable);
+      if (onlineTabletsForTable.isEmpty()) {
+        return result;
+      }
+      TabletId tabletId = onlineTabletsForTable.keySet().iterator().next();
```
Comment (keith-turner) [inline core/src/main/java/org/apache/accumulo/core/spi/balancer/SimpleLoadBalancer.java:271]: Can probably remove the `if (tabletId == null) { return result;}` a few lines down.  The new `if (onlineTabletsForTable.isEmpty()) {` a few lines above probably covers the case it was covering.

Comment (dlmarion) [reply core/src/main/java/org/apache/accumulo/core/spi/balancer/SimpleLoadBalancer.java:271]: Removed the other `if` in b82d5e3

**Comment by ctubbsii** [comment]
> I will create a follow-on task on merge to mark this method as deprecated in main.

Was this done?

**Comment by dlmarion** [comment]
> Was this done?

Done in #3977

**Pull request merged**